cmd/fossa/cmd/analyze/analyze.go

```diff
@@ -25,7 +29,8 @@ var Cmd = cli.Command{
 	Action:    Run,
 	ArgsUsage: "MODULE",
 	Flags: flags.WithGlobalFlags(flags.WithAPIFlags(flags.WithModulesFlags([]cli.Flag{
-		cli.BoolFlag{Name: flags.Short(Output), Usage: "show analysis output instead of uploading to FOSSA"},
+		cli.StringFlag{Name: flags.Short(Output), Usage: "send analysis to output file instead of uploading to FOSSA (default: -)"},
```

Comment:
I'd rather default to output on `STDOUT`. It seems like it should already be easy for a user to `fossa analyze --output > file`.

Reply:
using `-` to mean `STDOUT` seems to be a common practice with linux cli's per the POSIX guidlines http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap12.html#tag_12_02

```
For utilities that use operands to represent files to be opened for either reading or writing, the '-' operand should be used only to mean standard input (or standard output when it is clear from context that an output file is being specified).
```